### PR TITLE
fix(liveness): upgrade @aws-sdk/client-rekognitionstreaming to version patched with fast-xml-parser version 4.2.4

### PR DESCRIPTION
**Pull request by thaddmt**
… issue

#### Description of changes
- Fixes the fast-xml-parser security dependency from transitive dependency on fast-xml-parser
- Upgrades @aws-sdk/client-rekognitionstreaming used by the FaceLivenessDetector component
- Upgrades @aws-sdk/client-rekognition used by the Liveness example apps

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by thaddmt** [comment]
Smoke tested locally

File: .changeset/bright-dragons-smash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-liveness": patch
+---
+
+fix(liveness): upgrade aws sdk client-rekognitionstreaming to fix dep…
```
Comment (reesscot) [inline .changeset/bright-dragons-smash.md:5]: Can you update this to include the patch info?
```suggestion
fix(liveness): upgrade `@aws-sdk/client-rekognitionstreaming` to version patched with `fast-xml-parser` version `4.2.4`
```

**Review by sreeramsama — DISMISSED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -30032,7 +30052,7 @@ yallist@^4.0.0:
 
 yaml@2.2.1, yaml@2.2.2, yaml@^1.10.0, yaml@^1.10.2, yaml@^1.7.2, yaml@^2.1.1, yaml@^2.2.1:
   version "2.2.2"
-  resolved "https://registry.yarnpkg.com/yaml/-/yaml-2.2.2.tgz#ec551ef37326e6d42872dad1970300f8eb83a073"
+  resolved "https://registry.npmjs.org/yaml/-/yaml-2.2.2.tgz#ec551ef37326e6d42872dad1970300f8eb83a073"
```
Comment (reesscot) [inline yarn.lock:30055]: wish there was some way we could enforce either npmjs.com or yarnpkg.com

**Review by reesscot — APPROVED** [review]
LGTM assuming tests pass

**Review by zchenwei — APPROVED** [review]
LGTM

**Pull request merged**